tests/integration/test_vulnerability_detector/test_feeds/data/configuration_template/configuration_import_invalid_feed_type.yaml

```diff
@@ -15,7 +15,7 @@
               - os:
                   attributes:
                     - url: CUSTOM_FEED_URL
-                  value: '8'
```

Comment:
In this file, you have modified the configuration, however, if you do not change the test case, IT fails, because it keeps looking for the RHEL 8 log.

You need to modify the following file:
- https://github.com/wazuh/wazuh-qa/blob/8bb06c820c655d8a7b03898a5c740294d01ebe95/tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_import_invalid_feed_type.yaml#L8

Reply:
Done.